### PR TITLE
UBUNTU_LATEST does not work

### PR DESCRIPTION
**Pull request by xunpan**
1. The old code hardcoded docker version. But it does not exist now. Just removed and tested fine.
2. Make file change. There is no need to check and generate before build a binary. It wastes time for waiting this.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -149,13 +149,13 @@ generate_yaml_test: kubectl
 ############################################################
 build: manager clusterctl
 
-manager: depend generate check
```
Comment (gyliu513) [inline Makefile]: why did you remove `depend generate check`?

Comment (xunpan) [reply Makefile]: The related steps lasts for long time in building a binary. I think it is not necessary for testing a simple change in code for the binary.

Comment (gyliu513) [reply Makefile]: We cannot delete this, as the ` depend generate check` can make sure that we can always have correct format, and updated vendor files.

Comment (xunpan) [reply Makefile]: update code to make old target keep as is.

add new short target for quick build to accelerate working efficiency.
